tutorials/intermediate/color_cycle.py

```diff
@@ -39,8 +39,8 @@
 # cycler and a linestyle cycler by adding (``+``) two ``cycler``'s together.
 # See the bottom of this tutorial for more information about combining
 # different cyclers.
-default_cycler = cycler('color', ['r', 'g', 'b', 'y']) \
-                    + cycler('linestyle', ['-', '--', ':', '-.'])
+default_cycler = cycler(color=['r', 'g', 'b', 'y']) \
```

Comment:
can you replace the backslashes by properly aligned parenthetical forms too (below too)?